.travis.yml

```diff
@@ -44,6 +44,16 @@ jobs:
       env: BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_setfile=centos7-64 CHECK=beaker
       script: bundle exec rspec spec/acceptance/php56_spec.rb
       services: docker
+    - rvm: 2.5.3
+      bundler_args: --without development release
+      env: BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_setfile=ubuntu2004-64 CHECK=beaker
+      script: bundle exec rspec spec/acceptance/php_spec.rb
+      services: docker
+    - rvm: 2.5.3
```

Comment:
sorry for all the trouble :(
Puppet Inc. does not support Puppet 5 on Ubuntu 20.04. Can you remove this entry?